### ISSUE TITLE
AssumeRole with MFA

### ISSUE THREAD

**Issue by danielhamelberg** [aws-cloudformation/rain#7]
Can't assume roles that require MFA because AWS_SESSION_TOKEN not supported.

```
| => ./rain check -c --profile my_profile --debug
DEBUG: Couldn't load default config: token provider required for AssumeRole with MFA
DEBUG: AWS python: [/usr/bin/python3]
- Loading AWS config...DEBUG: Cred dumper output: {"AWS_ACCESS_KEY_ID": "[REDACTED]", "AWS_SECRET_ACCESS_KEY": "[REDACTED]", "AWS_SESSION_TOKEN": null, "AWS_REGION": "eu-west-1"}

DEBUG: Loaded credentials from cred dumper
Unable to load identity: The security token included in the request is invalid
```

**Comment by stilvoid** [comment]
Thanks for filing this with the detail :) I'll look into this soon.

**Comment by stilvoid** [comment]
Fixed in 6d84c17. I'll publish a new version shortly.
